Sources/Commands/SwiftBuildTool.swift

```diff
@@ -13,7 +13,6 @@ import Basics
 import Build
 import PackageGraph
 import SPMBuildCore
-import TSCBasic
```

Comment:
By a rough count, there are about 10% of changes to `import`s not because of the changes to `Basics`. Would it be better to separate them into their own PR, to reduce the amount of files changed in this PR? Although, the PR probably still would contain changes to about 200 files.

Reply:
exactly, this is why I ended up doing together since it would anyways be a huge PR